python_tests/pytest_input_generators.py

```diff
@@ -13,6 +13,62 @@
 from nvfuser import DataType
 
 
+def cat_generator(
+    op: OpInfo, dtype: torch.dtype, requires_grad: bool = False, **kwargs
+):
+    make_arg = partial(
+        make_tensor, device="cuda", dtype=dtype, requires_grad=requires_grad
+    )
+    # shapes, dim
+    cases = [
+        ([(3,)], 0),  # single tensor provided
+        # 1D
+        ([(2,), (3,)], 0),
+        ([(2,), (4,)], 0),
+        ([(0,), (2,)], 0),
+        ([(0,), (2,)], -1),
+        ([(2, 3), (2, 4)], 1),
+        ([(2, 3), (2, 4), (2, 5)], 1),
```

Comment:
Could you put a comment here about size one and link to issue https://github.com/NVIDIA/Fuser/issues/224?